requirements.txt

```diff
@@ -26,11 +26,11 @@ aiofiles==22.1.0
 
 # extras
 aws-secretsmanager-caching==1.1.1.5
-boto3==1.24.19
-botocore==1.27.19
+boto3
+botocore
 db-dtypes==1.0.5
 google-api-python-client==2.70.0
-google-cloud-bigquery~=2.34.4
+google-cloud-bigquery~=3.0
```

Comment:
Please make sure the package version here is compatible with package version in mage_integration requirements.txt https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/requirements.txt#L9

Reply:
K